doc/packaging-guide.xml

```diff
@@ -1,56 +1,77 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
          xmlns:xlink="http://www.w3.org/1999/xlink"
-         xml:id="chap-quick-start">
+         xml:id="chap-packaging-guide">
 
-<title>Quick Start to Adding a Package</title>
+  <title>Packaging Guide</title>
 
-<para>To add a package to Nixpkgs:
+  <section xml:id="sec-policies">
 
-<orderedlist>
+    <title>Policies</title>
+
+    <para>The forum for policy discussion can be found at the <link
+xlink:href="https://github.com/NixOS/nixpkgs/labels/6.topic%3A%20policy%20discussion">
+issue tracker, topics are labeled with 'policy
+discussion'</link>. This section brewed from maintainer
+opinions. Regard the following paragraphs as drafts.</para>
+
+    <para>The nixpkgs package repository is not restrictive in
+comparison to other distributions. Non free packages are allowed but
+will not be built by Hydra. Packaging binaries is allowed, patching
```

Comment:
The first sentence in this paragraph I would drop since it doesn't add anything really.

Next, you will have to make a clearer distinction between free and non-free software. Indeed, we package non-free software and then we typically need to patch the binaries.

In the case of free software it is different. Here we want all software to be build from source. There are cases that we do use binaries, but those are, at least how I see it, _exceptions_, and they are typically only included because of, say, a complex build procedure.




> Wh